packages/script/test/script.test.ts

```diff
@@ -16,9 +16,9 @@ import { generateTestWallet } from '@fuel-ts/wallet/test-utils';
 import { readFileSync } from 'fs';
 import { join } from 'path';
 
-import { jsonAbiMock, jsonAbiFragmentMock } from '../test/fixtures/mocks';
+import { Script } from '../src/script';
 
-import { Script } from './script';
+import { jsonAbiMock, jsonAbiFragmentMock } from './fixtures/mocks';
```

Comment:
Could you please keep the `test/script.test.ts` side-by-side with its source `src/script.ts` counterpart?

The sway stuff is better placed in the `test` directory, as you did already. Good call. 👌

Reply:
I've moved it to the `test` folder following the example of `providers/test/provider.test.ts`. Sure, I can move it back, but it feels wrong to me that the `src` folder imports stuff from the `test` folder (`import { jsonAbiMock, jsonAbiFragmentMock } from '../test/fixtures/mocks'`). It should be the other way around, that the `test` folder imports files from `src/index` that it wants to test (`import { Script } from '../src/index'`). The whole testing of the public API thing.

I've opened up a discussion #1290 where I'll summarize my thoughts on this issue. @danielbate might be interested as well. Maybe there were already _(finished?)_ discussions on GH about it that I don't know about? If so, please let me know.